vaas-app/src/vaas/vcl/templates/vcl_blocks/4.0/RECV.tvcl

```diff
@@ -6,6 +6,11 @@ sub vcl_recv {
     # Call protocol redirect sub
     call protocol_redirect;
 
+    # Handler for no backend in director
+    if(req.http.x-action == "nobackend") {
+        return(synth(404, "<!--Director " + req.http.x-director + " has no backends or is disabled-->"));
```

Comment:
I suppose after that change - replacing tag <SET_BACKEND_*> in renderer.py#344 is uneccessary

Reply:
that's true. I will change this.